### ISSUE TITLE
Accessibility update for Menubar Vertical Navigation

### ISSUE THREAD

**Issue by ethib137** [liferay/clay#5170]
## What is your proposal?
To improve the accessibility of the Menubar it should be based off of the [W3 Menu Bar pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menu/).

It should include role="menubar" with aria-orientation="vertical" and menus within that have role="menu" and items with role="menuitem".

This would provide better keyboard functionality, allowing the movement through menubar, menu, and menu items using only arrow keys instead of tab. 

## Why would adopting this proposal be beneficial?

Improved accessibility.

## Needs to be discussed
How should the arrow keys operate moving between top menubar items and submenus.

cc: @marcoscv-work

**Comment by matuzalemsteles** [comment]
Hey @ethib137 just to confirm but you are referring to that component https://storybook.clayui.com/?path=/story/design-system-components-nav--vertical-nav, right?

**Comment by marcoscv-work** [comment]
Yeah, the idea is use the component or the component reference to apply patterns to Product menu

**Comment by ethib137** [comment]
Yes, that's correct. :)
